frontend/src/components/BikeGallery/BikeGallery.tsx

```diff
@@ -1,26 +1,23 @@
-import React from 'react';
-import { Bike } from '../../models/Bike';
-import BikeCard from '../BikeCard/BikeCard';
-import AddBike from "../AddBike/AddBike";
+import React from 'react'
+import { Bike } from '../../models/Bike'
+import BikeCard from '../BikeCard/BikeCard'
+import AddBike from '../AddBike/AddBike'
 
 type BikeGalleryProps = {
    bikes: Bike[]
-
-   addBike: (newBikeTitle: string) => Promise<void>
+   addBike: (newBikeTitle: string) => void
 }
 
 function BikeGallery(props: BikeGalleryProps) {
    return (
-       <>
+      <>
```

Comment:
Is this fragment necessary?

Reply:
Yes, because a React component may only return a SINGLE component (or fragment)